ui/src/app/cohort-search/multi-select/multi-select.component.ts

```diff
@@ -14,7 +14,7 @@ export class MultiSelectComponent implements OnInit, OnDestroy {
   @Input() includeSearchBox = true;
   @Input() options = List();
   @Input() set initialSelection(opts) {
-    const _selections = opts.map(opt => opt.hashCode()).toSet();
+    const _selections = opts.map(opt => opt.get('parameterId')).toSet();
```

Comment:
Regression test? This seems like a subtle bug and though I'm still trying to understand the code better I'm not sure I would have caught this in code review.

Reply:
Tests are added, at least for this aspect.